### ISSUE TITLE
Network error undefined from ApolloError

### ISSUE THREAD

**Issue by jessicavaughan820** [fubhy/graphql-transport-electron#1]
I'm getting the following error when try to run my Electron app:

`Uncaught (in promise) Error: Network error: undefined
    at new t (ApolloError.ts:57)
    at QueryManager.ts:576
    at QueryManager.ts:1091
    at Set.forEach (<anonymous>)
    at QueryManager.ts:1087
    at Map.forEach (<anonymous>)
    at e.broadcastQueries (QueryManager.ts:1085)
    at QueryManager.ts:434`

Do you have any additional instructions on how to implement this module?
Thanks in advance.

**Comment by pellejacobs** [comment]
Same issue here. @fubhy do you have a minimal example that implements the library?

**Comment by pellejacobs** [comment]
It seems the issue is because javascript Errors can't be serialised. As a result, Error's can't be transmitted over ipc as is: https://github.com/electron/electron/issues/7956. 

A workaround involves manually serialising the error here: https://github.com/fubhy/graphql-transport-electron/blob/master/src/server.ts#L77. @fubhy would you accept a PR for this change?

**Comment by fubhy** [comment]
@pellejacobs Absolutely. :-)

**Comment by fubhy** [comment]
The graphql errors usually also contain stacktraces, references to the graphql schema, types etc. We might want to simply drop those in this case for simplicity.

**Comment by Michaelp1994** [comment]
I had the same issue as above and found `createSchemaLink` does not work without providing a context. 

Example of working main process: 

```
const typeDefs = 
    type Query {
        test: String!
    };
    const resolvers = {
      Query: {
        test: async (parent, args, context) => {
          console.log("RECVD");
          return "TEST";
        }
      }
    };
    const schema = makeExecutableSchema({
      typeDefs,
      resolvers
    });

    const link = createSchemaLink({
      schema,
      context: async request => {
        // console.log(request);
      }
    });
```

**Comment by fubhy** [comment]
Good catch

**Comment by fubhy** [comment]
I released a new version (1.0.3) that should fix this.
